influunt-app/app/styles/influunt.css

```diff
@@ -248,10 +248,6 @@
   background: transparent;
 }
 
-.ibox-title i {
-  color: #1ab394;
-}
```

Comment:
Aqui também, essa regra foi apagada no `.css` mas continua no `.scss`, quando o APP for rodar essa regra vai ser colocada de volta nesse arquvo `.css`.